src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final String defaultFigureColor = Color.WHITE.name();
+    public static final double defaultFigureRadius = 10.0;
+
+    private Square squareObject() {
+        return new Square(new ColorSupplier().getRandomColor(),
+                new Random().nextInt(10));
```

Comment:
[All magic numbers in your code should be constants](https://github.com/petro2061/jv-oop-advanced/blob/myOopAdvanced/checklist.md#all-magic-numbers-in-your-code-should-be-constants)